### PR TITLE
Add NCyte Full name

### PR DESCRIPTION
**Pull request by JosueMolinaMorales**
This PR completes [8565](https://app.shortcut.com/clarkcan/story/8565/add-full-name-to-ncyte-collection-card)

Screen Shot 2022-04-04 at 4 25 50 PM

### CODE REVIEW

**Review by EVogel1999 — APPROVED** [review]

**Review by StarChaser44 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]
Looks awesome! Just hold off on merging until I get their approval on this.

**Review by mfranz1 — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Pull request merged**
